app/importers/cma_import.rb

```diff
@@ -32,7 +32,11 @@ def import_job_builder
   end
 
   def attachment_attacher
-    CmaImportAttachmentAttacher.new(attribute_mapper, data_files_dir)
+    CmaImportAttachmentAttacher.new(
+      attribute_mapper,
```

Comment:
This would probably benefit from being made to use keyword arguments now, but that's not something that should stop us getting this merged